backend/src/main/java/com/benine/backend/camera/CameraController.java

```diff
@@ -44,6 +42,22 @@ public void addCamera(Camera camera) {
     highestIdInUse++;
     cameras.add(camera);
   }
+  
```

Comment:
I was afraid things could get confusing when calling this method from other classes so making them private would be a perfect solution :)